src/main/java/FrameworksDrivers/CompareGraphics.java

```diff
@@ -0,0 +1,136 @@
+package FrameworksDrivers;
+
+import Entities.Person;
+import Entities.Schedule;
+import Entities.events.Event;
+import InterfaceAdapters.ScheduleManager;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.JFrame;
+import java.util.Objects;
+
+public class CompareGraphics extends JFrame implements ActionListener {
+    private Graphics g2g;
+    private ScheduleManager manager;
+    private Person user1;
+    private Person user2;
+
+    private JLabel titleLabel;
+
+    private String[] timesList = {"00:00", "01:00", "02:00", "03:00", "04:00", "05:00", "06:00", "07:00", "08:00", "09:00", "10:00", "11:00", "12:00", "13:00", "14:00", "15:00", "16:00", "17:00", "18:00", "19:00", "20:00", "21:00", "22:00", "23:00"};
+
+
+    public CompareGraphics(Person user1, Person user2, ScheduleManager manager) {
```

Comment:
input user names, use schedule manager's userlist instance attribute to avoid importing Person

Reply:
this is already so much code though, we should probably just merge it in and fix it afterward